challenge_03_loops-conditionals/loops-conditionals/while/exercise_02.js

```diff
@@ -0,0 +1,56 @@
+/*
+    1. Create a while loop that runs from 1-100
+    2. Every 10 loops, print out: Checkpoint! and the loop number
+    3. On loop 50 print out ONLY: Half way there!
+    4. On the final loop (100) print out ONLY: You made it!
+    5. AFTER the loop is done, print out: All, done!
+
+    Expected output:
+    Checkpoint! 10
+    Checkpoint! 20
+    Checkpoint! 30
+    Checkpoint! 40
+    Half way there!
+    Checkpoint! 60
+    Checkpoint! 70
+    Checkpoint! 80
+    Checkpoint! 90
+    You made it!
+    All, done!
+*/
+
+/**
+ * isTenthWhile
+ */
+const isTenthWhile = function (startNumber, limitNumber) {
+  let currentNumber = startNumber;
+
+  while (currentNumber <= limitNumber) {
+    if (currentNumber !== 100) {
```

Comment:
Omg Marc, why is there an exclamation mark you stupid dumb fucking retard. Fix this immediately
```suggestion
    if (currentNumber == 100) {
```

Reply:
@ZilongZhan , same here.